src/command_line_parser.h

```diff
@@ -155,15 +139,11 @@ struct PerfAnalyzerParameters {
   // The profile export file path.
   std::string profile_export_file{""};
 
-  bool is_using_periodic_concurrency_mode{false};
   Range<uint64_t> periodic_concurrency_range{1, 1, 1};
   uint64_t request_period{10};
   size_t warmup_request_count{0};
 
-  std::vector<float> schedule{};
```

Comment:
As mentioned, the fixed schedule source of truth is now in the input data JSON `timestamp` fields instead of being passed in via `--schedule=<n>,<n>,...`.